### DISCUSSION TITLE
22621.3880.66.7, start menu right click not working

### DISCUSSION THREAD

**Discussion by dawidmosk** [valinet/ExplorerPatcher#3865] [General]
Anyone have solution for it? Reset, uninstall not working.
Right click on icons, clock etc. are working fine but on start menu - where is menu with Computer Management, Device manager, Disk management etc. is not.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3837

please close this.

**Reply by dawidmosk** [reply]
Thx
